app/proc/telegram.go

```diff
@@ -125,15 +144,53 @@ func (client TelegramClient) sendAudio(channelID string, item feed.Item) (*tb.Me
 		Title:    item.Title,
 	}
 
-	message, err := audio.Send(
+	return audio.Send(
 		client.Bot,
 		recipient{chatID: channelID},
 		&tb.SendOptions{
 			ParseMode: tb.ModeHTML,
 		},
 	)
+}
 
-	return message, err
+func (client TelegramClient) sendAudioWithExternalUploader(channelID string, item feed.Item) (*tb.Message, error) {
+	httpBody, err := client.downloadAudio(item.Enclosure.URL)
+	defer httpBody.Close() //nolint:staticcheck
+	if err != nil {
+		return nil, err
+	}
+
+	tmpFile, err := ioutil.TempFile(os.TempDir(), "feed-master-")
+	if err != nil {
+		return nil, errors.Wrap(err, "create temoprary file")
+	}
+	defer os.Remove(tmpFile.Name())
+
+	_, err = io.Copy(tmpFile, httpBody)
+	if err != nil {
+		return nil, errors.Wrap(err, "write to temporary file")
+	}
+
+	output, err := execute(
+		"python3 "+client.uploaderConfig.PathToScript,
+		map[string]string{
```

Comment:
the idea of calling external process and pass params as env feels a little bit unusual. Is there a good reason not to use the usual cli params, i.e. --key=val

Reply:
Done
Originally wanted to follow twelve-factor methodology